src/fsharp/symbols/Symbols.fs

```diff
@@ -2147,6 +2147,18 @@ and FSharpAssemblySignature private (cenv, topAttribs: TypeChecker.TopAttribs op
             tA.assemblyAttrs
             |> List.map (fun a -> FSharpAttribute(cenv,  AttribInfo.FSAttribInfo(cenv.g, a))) |> makeReadOnlyCollection
 
+    member __.FindEntityByPath path =
+        let inline findNested name = function
```

Comment:
can you please inline it manually? I don't think the local function is bringing any advantage